pkg/services/baremetal/host/host.go

```diff
@@ -1600,6 +1600,7 @@ func (s *Service) handleCloudInitNotStarted() actionResult {
 		if err := handleSSHError(out); err != nil {
 			return actionError{err: fmt.Errorf("failed to reboot: %w", err)}
 		}
+		createSSHRebootEvent(s.scope.HetznerBareMetalHost, "CheckCloudInitLogsForSigTerm failed")
```

Comment:
I don't get this. This is not about ssh reboot is it?

Reply:
@janiskemper 

Some lines above: `out = oldSSHClient.Reboot()`. 

We could give that a better name, but overall the reboot gets done because the output of `CheckCloudInitLogsForSigTerm` is not the empty string.

Reply:
I made a commit